### ISSUE TITLE
Dewikification: Make an app to serve guides

### ISSUE THREAD

**Issue by lemonsaurus** [python-discord/site#383]
**Remember to target the `dewikification` branch with this pull request**

Create a new Django app that will serve guides. 

This app will have a folder full of markdown files with guide content, and will render those markdown files dynamically depending on the url. Something like `pythondiscord.com/guides/<markdown_file_name>` will navigate to a page displaying all the content in the correct markdown file. These markdown files should be GitHub-flavored markdown.

## What should a page like this look like?
Well, it should look like one of our current guides.
![image](https://user-images.githubusercontent.com/2098517/92272959-cbf7e400-eeea-11ea-878b-840e62db26ce.png)

Basically just a navbar, breadcrumbs, the content of the markdown file, and a footer.

**Comment by ks129** [comment]
I will work on it

**Comment by kosayoda** [comment]
Done :tada:
